src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -950,6 +969,13 @@ def _add_plane_segmentation(
             data=is_id_rejected,
         )
 
+    # Always add quality metrics if they are available
```

Comment:
I think the default definitions should be here:

```
    quality_metrics_definitions = {
        "snr": "Signal-to-noise ratio for each component",
        "r_values": "Spatial correlation values for each component",
        "cnn_preds": "CNN classifier predictions for component quality",
    }
```

close to where they are used.